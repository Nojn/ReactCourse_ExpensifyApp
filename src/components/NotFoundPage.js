import React from "react";
import { BrowserRouter as Router, Link} from 'react-router-dom';

const NotFoundPage =() => (
    <div>
        404! 
        <Link to='/'>Go to homepage</Link>
    </div>
);

export default NotFoundPage;