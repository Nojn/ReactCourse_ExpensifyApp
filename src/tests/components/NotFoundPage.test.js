import React from 'react';
import { shallow } from 'enzyme';
import ReactShallowRendering from 'react-test-renderer/shallow';
import NotFoundPage from '../../components/NotFoundPage.js';



test ('should render 404 page correctly', ()=>{
    const wrapper = shallow(<NotFoundPage/>);
    expect(wrapper).toMatchSnapshot();
   
});