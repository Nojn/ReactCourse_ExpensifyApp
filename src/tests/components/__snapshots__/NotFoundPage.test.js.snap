// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404 page correctly 1`] = `ShallowWrapper {}`;
